config/dataset/test_ci_segmentation_dataset.yaml

```diff
@@ -0,0 +1,17 @@
+# @package _global_
+dataset:
+  name:
+  modalities: RGB
+  input_dim: 256
+  overlap:
+  min_annot_perc:
+  raw_data_csv: ${general.raw_data_csv}
+  raw_data_dir: ${general.raw_data_dir}
+  sample_data_dir: ${general.sample_data_dir}
+  class_name: # will follow in the next version
+  classes_dict: {'Building':1}
+  class_weights:
+  train_val_percent: {'trn':0.7, 'val':0.2, 'tst':0.1}
```

Comment:
Test samples are currently determined by the dataset (trn or tst) assigned in the dataset csv. Why add 'tst' here? Is it more of a TODO than an actual change we should merge?